lib/virtual-fs/src/overlay_fs.rs

```diff
@@ -1688,4 +1529,153 @@ mod tests {
             FsError::EntryNotFound
         )
     }
+
+    // OLD tests that used WebcFileSystem.
+    // Should be re-implemented with WebcVolumeFs
+    // #[tokio::test]
+    // async fn wasi_runner_use_case() {
```

Comment:
Is it possible to reuse this test for `webc_volume_fs`?

Reply:
Probably, but we can do it in a PR apart